src/Installers/Windows/SharedFrameworkBundle/1040/thm.wxl

```diff
@@ -27,38 +27,38 @@
   <String Id="OptionsLocationLabel">Percorso di installazione:</String>
   <String Id="OptionsBrowseButton">&amp;Sfoglia</String>
   <String Id="OptionsOkButton">&amp;OK</String>
-  <String Id="OptionsCancelButton">&amp;Annulla</String>
+  <String Id="OptionsCancelButton">Ann&amp;ulla</String>
```

Comment:
@wtgodbe and @joeloff am I correct that an ampersand in a .wxl file identifies the next character as an accelerator key❔ If yes, is this changed translation an improvement❔

@cristianosuzuki77 what are the heuristics the translators use to decide where to place the accelerator key❔

Reply:
I found a few mentions of accelerator / accessibility keys in WiX and it seems the toolkit follows the conventions of other Windows installers. An ampersand identifies the key.

The bundle already has mid-word accelerators in a few strings e.g. https://github.com/dotnet/aspnetcore/pull/43956/files#diff-ff840b97875d2e933bba46b993fed4078e0fb5b6abd097e87d7b8a4b10b0e517R22 (though none in the English strings that I noticed).

So, is changing the accelerator key from `A` to `u` a good idea❔ It's consistent across `Cancel` buttons in this language and a few other accelerator keys are changing e.g. https://github.com/dotnet/aspnetcore/pull/43956/files#diff-4d0161edbccd7dd3b1576f2c6530fb898b0dc60e46169d1082700cc89e04b81dR36 and <https://github.com/dotnet/aspnetcore/pull/43956/files#diff-5be1b04c80c351d6cb288862f266fbc0e6e82eeed53d0094a9979c48d7935072R54>. I am just not sure why and hope to hear from @cristianosuzuki77.

Reply:
Opened https://ceapex.visualstudio.com/CEINTL/_workitems/edit/709046

Reply:
This has been resolved. Leaving it open only to provide fodder for a release note. /cc @ldillonel